static-site/src/util/blogPosts.js

```diff
@@ -36,6 +42,76 @@ export function getBlogPosts() {
   return blogPosts;
 }
 
+/**
+ * Writes RSS2, Atom, and JSON feeds into the `public/blog` directory.
+ */
+export async function generateBlogFeeds() {
+  const feedDir = `${__dirname}/../../public/blog`;
+  const siteURL = "https://nextstrain.org/blog";
+
+  const feed = new Feed({
+    title: "Nextstrain.org Blog",
+    description: "FIXME",
+    id: siteURL,
+    link: siteURL,
+    image: "https://nextstrain.org/_next/static/media/nextstrain-logo-small.dcf6bdd2.png",
```

Comment:
```suggestion
    image: "https://nextstrain.org/nextstrain-logo-small.png",
```

Use `static-site/public/nextstrain-logo-small.png` which is served at the top level using the filename as-is, not `static-site/static/logos/nextstrain-logo-small.png` which is bundled by webpack with a hash in the filename that may change over time.